index.html

```diff
@@ -927,6 +927,9 @@
                 <li>If the pointer event dispatched was <code>pointerup</code> or <code>pointercancel</code>, clear the <i>PREVENT MOUSE EVENT</i> flag for this <code>pointerType</code>.</li>
             </ol>
             <div class="note">
+                <p>If the user agent supports both Touch Events (as defined in [[TOUCH-EVENTS]]) and Pointer Events, generation of compatibility mouse events as described in this section is not very web compatible. In this scenario gesture based generation of compatiblity mouse events for primary PointerEvents is better. For more information see [[TOUCH-EVENTS]].</p>
```

Comment:
assuming this is aimed at UA developers, rather than content developers? "generation of compatibility mouse events as described in this section is not very web compatible" sounds a little off stylistically, and doesn't give me quite a clear indication of what we're trying to say. something more like "the user agent SHOULD NOT generate compatibility mouse events as described in this section, as this is likely to introduce compatibility problems for sites that expect mouse events to be generated in accordance with the model outlined in [[TOUCH-EVENTS]]." and then drop the last "For more info..." sentence?